### ISSUE TITLE
Going to standby crashes gnome-shell

### ISSUE THREAD

**Issue by traschke** [LinxGem33/Arc-Menu#212]
Hi,

with Arc-Menu installed and activated I cannot put my computer to standby or hibernate anymore, without crashing the current gnome-shell session.

In the process of going to sleep the shell crashes and the login-screen (gdm) is visible for several seconds, before the system goes into standby mode. After powering the device back on and logging back into my account, a new session is started.

I am using the current version from the source code of this repo.

Distrubution:
`Arch Linux 4.14.13-1-ARCH`

Gnome Shell:
`GNOME Shell 3.26.2`

Display-Server:
`Type=wayland`

Logs:
```
Key repeat discarded, Wayland compositor doesn't seem to be processing events fast enough!
Key repeat discarded, Wayland compositor doesn't seem to be processing events fast enough!
Key repeat discarded, Wayland compositor doesn't seem to be processing events fast enough!
Key repeat discarded, Wayland compositor doesn't seem to be processing events fast enough!
Key repeat discarded, Wayland compositor doesn't seem to be processing events fast enough!
Key repeat discarded, Wayland compositor doesn't seem to be processing events fast enough!
Key repeat discarded, Wayland compositor doesn't seem to be processing events fast enough!
Key repeat discarded, Wayland compositor doesn't seem to be processing events fast enough!
Key repeat discarded, Wayland compositor doesn't seem to be processing events fast enough!
Key repeat discarded, Wayland compositor doesn't seem to be processing events fast enough!
Window manager warning: Failed to set power save mode for output eDP-1: Keine Berechtigung
Screen lock is locked down, not locking
gsignal.c:2641: instance '0x55f7e8c71650' has no handler with id '44038'
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
clutter_layout_manager_get_child_meta: assertion 'CLUTTER_IS_LAYOUT_MANAGER (manager)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
g_object_set: assertion 'G_IS_OBJECT (object)' failed
(EE)
Fatal server error:
(EE) failed to read Wayland events: Connection reset by peer
(EE)
```

**Comment by LinxGem33** [comment]
hi appreciate the feed back

are you still having this same issue as of `kernel 4.14.15` on arch.

**Comment by traschke** [comment]
Yes, still the same issue with `4.14.15-1-ARCH`, `GNOME Shell 3.26.2` on wayland.
I've took another log with above configuration plus all gnome extensions disabled, except arc-menu.
The file is relatively long, so I uploaded it to pastebin: https://pastebin.com/NgadJGnL

**Comment by LinxGem33** [comment]
going to set this issue with a medium priority as it seems to be related to these issues and also seems to be affecting Arch based distros running gnome shell 3.26.2, will be logged for further investigation.

#215  #212 #209

**Comment by LinxGem33** [comment]
this issue should now be fixed #218 please install from the latest master branch and let me know if it has indeed fixed the issue on your systems.

**Comment by traschke** [comment]
It is fixed for me.
